concurrent.lib.sh

```diff
@@ -144,6 +144,12 @@ concurrent() (
     __crt__ORIG_BASHOPTS=${BASHOPTS}
     __crt__ORIG_SHELLOPTS=${SHELLOPTS}
 
+    if [[ `uname` == 'Darwin' ]]; then
+        __crt__sed='gsed'   # for OS X
+    else
+        __crt__sed='sed'
+    fi
+
```

Comment:
Thank you for reviewing it!
I support your idea.

Since sed is used in __crt__help__display_version_and_exit, I think the "Compatibility Check" may be placed at top of the concurrent().